src/sidebar/components/sidebar-content.js

```diff
@@ -297,17 +312,26 @@ function SidebarContentController(
   this.focus = focusAnnotation;
   this.scrollTo = scrollToAnnotation;
 
-  this.selectedAnnotationCount = function() {
+  this.annotationsAreSelected = function() {
```

Comment:
I'm kinda thinking this should maybe even be isAnnotationSelected because right now you can only 'select' one annotation at a time. I'm guessing this was intended for sometime in the future when we can select multiple annotations but who knows when that's gonna be-might be simpler to just name it according to what we have right now.